Makefile

```diff
@@ -474,6 +474,28 @@ gen_genesis_and_config:
 clear_genesis_and_config:
 	rm build/config/gen.*.json
 
+.PHONY: localnet_up
+## Starts localnet with all dependencies (basically, `tilt up`)
+localnet_up:
+	tilt up
+
+.PHONY: localnet_client_debug
+## Starts debug client (allows to control LocalNet)
+localnet_client_debug:
+	kubectl exec -it deploy/pocket-v1-cli-client -- client debug
+
+.PHONY: localnet_shell
+## Starts debug client (allows to control LocalNet)
+localnet_shell:
+	kubectl exec -it deploy/pocket-v1-cli-client -- /bin/bash
+
+.PHONY: localnet_down
+## Stops localnet and cleans up dependencies (basically, `tilt down` + resources might not be cleaned up by tilt)
+localnet_down:
+	tilt down
+	# kubectl delete --all --namespace=default pocketvalidators
+	kubectl get sts -n default --no-headers=true | awk '/v1-validator/{print $1}' | xargs kubectl delete -n default  sts
+
```

Comment:
I suppose so, alternatively I suspect a change might be done in a Tiltfile to do this. I've looked into this only briefly and decided to change a make target.

This is only necessary because of our set-up with operator, which controls only some resources, when tilt controls the rest of them.